tests/unit/components/search/MoreFiltersDropdown.spec.js

```diff
@@ -1,16 +1,26 @@
 import { createLocalVue, mount } from '@vue/test-utils';
 import BootstrapVue from 'bootstrap-vue';
-import MoreFacetsDropdown from '../../../../components/search/MoreFacetsDropdown.vue';
+import MoreFacetsDropdown from '../../../../components/search/MoreFiltersDropdown.vue';
 
 const localVue = createLocalVue();
 localVue.use(BootstrapVue);
 
+const $store = {
```

Comment:
The Vuex store should not be initialised as a plain object nor included in $mocks. Do this instead:
```diff
diff --git a/tests/unit/components/search/MoreFiltersDropdown.spec.js b/tests/unit/components/search/MoreFiltersDropdown.spec.js
index 74dce04..f478631 100644
--- a/tests/unit/components/search/MoreFiltersDropdown.spec.js
+++ b/tests/unit/components/search/MoreFiltersDropdown.spec.js
@@ -1,18 +1,19 @@
 import { createLocalVue, mount } from '@vue/test-utils';
 import BootstrapVue from 'bootstrap-vue';
 import MoreFacetsDropdown from '../../../../components/search/MoreFiltersDropdown.vue';
+import Vuex from 'vuex';
 
 const localVue = createLocalVue();
 localVue.use(BootstrapVue);
+localVue.use(Vuex);
 
-const $store = {
+const store = new Vuex.Store({
   state: {
     entity: {
       id: null
     }
   }
-};
-
+});
 
 const factory = () => mount(MoreFacetsDropdown, {
   localVue,
@@ -20,11 +21,11 @@ const factory = () => mount(MoreFacetsDropdown, {
     selected: {}
   },
   mocks: {
-    $store,
     $t: (key) => key,
     $tc: (key) => key,
     $te: (key) => key
-  }
+  },
+  store
 });
 
 describe('components/search/MoreFacetsDropdown', () => {

```